lib/media/time_ranges_utils.js

```diff
@@ -144,19 +144,9 @@ shaka.media.TimeRangesUtils = class {
       return null;
     }
 
-    // Some browsers will stop earlier than others before a gap (e.g. IE/Edge
-    // stops 0.5 seconds before a gap). So for some browsers we need to use a
-    // larger threshold. See: https://bit.ly/2K5xmJO
-    const useLargeThreshold = Platform.isEdge() ||
-                              Platform.isIE() ||
-                              Platform.isTizen() ||
-                              Platform.isChromecast();
-
-    const threshold = useLargeThreshold ? 0.5 : 0.1;
-
     const idx = TimeRangesUtils.getBufferedInfo(b).findIndex((item, i, arr) => {
       return item.start > time &&
-          (i == 0 || arr[i - 1].end - time <= threshold);
+          (i == 0 || arr[i - 1].end - time <= config.gapThreshold);
```

Comment:
It would be better to only pass the `gapThreshold` value instead of the whole config.  Also please update the function description to include the new argument.